### PR TITLE
init

### PR DESCRIPTION
**Pull request by smiller171**

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2019 Rhythmic Technologies, Inc.
```
Comment (sblack4) [inline LICENSE]:
```suggestion
Copyright (c) 2020 Rhythmic Technologies, Inc.
```

**Review by sblack4 — CHANGES_REQUESTED** [review]
This looks way easier than packer. One small date change though

File: LICENSE (hunk 1 above)
Comment (smiller171) [reply LICENSE]: Ultimately this is simpler only because it's less flexible, but it's more specialized to our needs.

**Review by sblack4 — APPROVED** [review]
:shipit:

File: variables.tf [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+  type        = string
+}
+
+variable "tags" {
+  default     = {}
+  description = "map of tags to use for CFN stack and component"
+  type        = map(string)
+}
+
+variable "timeout" {
```
Comment (cdaniluk) [inline variables.tf]:
```suggestion
variable "cloudformation_timeout" {
```

File: main.tf [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+    playbook_dir  = var.playbook_dir
+    playbook_file = var.playbook_file
+    playbook_repo = var.playbook_repo
+  })
+}
+
+resource "aws_cloudformation_stack" "this" {
+  name               = var.name
+  on_failure         = "ROLLBACK"
+  timeout_in_minutes = var.timeout
```
Comment (cdaniluk) [inline main.tf]:
```suggestion
  timeout_in_minutes = var.cloudformation_timeout
```

File: main.tf [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+    uri                = var.data_uri
+    version            = var.component_version
+
+    tags = merge(
+      var.tags,
+      { Name : var.name }
+    )
+  })
+
+  tags = merge(
```
Comment (cdaniluk) [inline main.tf]: sort above `template_body`

**Review by cdaniluk — CHANGES_REQUESTED** [review]
proforma comment about the output name and lazyCamelCase from the other PRs. Whatever we decide we should do it consistently across all 3 of these modules.

**Comment by sblack4** [comment]
> proforma comment about the output name and lazyCamelCase from the other PRs. Whatever we decide we should do it consistently across all 3 of these modules.

Lettuce mark this decision in the annals of engineering standards, https://github.com/rhythmictech/terraform-aws-imagebuilder-recipe/pull/1#issuecomment-636867377

File: variables.tf [hunk 5]
```diff
@@ -0,0 +1,68 @@
...
+  type        = string
+}
+
+variable "tags" {
+  default     = {}
+  description = "map of tags to use for CFN stack and component"
+  type        = map(string)
+}
+
+variable "cloudformation_timeout" {
```
Comment (sblack4) [inline variables.tf]: Alphabetization

**Comment by smiller171** [comment]
> > proforma comment about the output name and lazyCamelCase from the other PRs. Whatever we decide we should do it consistently across all 3 of these modules.
> 
> Lettuce mark this decision in the annals of engineering standards, [rhythmictech/terraform-aws-imagebuilder-recipe#1 (comment)](https://github.com/rhythmictech/terraform-aws-imagebuilder-recipe/pull/1#issuecomment-636867377)

unmark your PR as a draft so we can review it

**Review by cdaniluk — APPROVED** [review]

**Pull request merged**